src/runtime/instruction_info.zig

```diff
@@ -85,9 +85,8 @@ pub const InstructionInfo = struct {
         self: *const InstructionInfo,
         pubkey: Pubkey,
     ) bool {
-        for (self.account_metas.constSlice()) |account_meta| {
+        for (self.account_metas.slice()) |account_meta|
```

Comment:
Updated while resolving the merge conflict when rebasing. Will revert.

Reply:
[c4ecc1c](https://github.com/Syndica/sig/pull/576/commits/c4ecc1cbbbdd2ddae69f194deed3078f31836dd2)